src/helpers/product-helper.php

```diff
@@ -35,6 +35,6 @@ public function get_name() {
 	 * @return bool True when is premium.
 	 */
 	public function is_premium() {
-		return \defined( 'WPSEO_PREMIUM_PLUGIN_FILE' );
+		return \defined( 'WPSEO_PREMIUM_FILE' );
```

Comment:
Shouldn't this be renamed also in the other two occurrences in the codebase
- the now unused `is_yoast_seo_premium` method (could still be used by add-ons, to be checked)
- the `test_get_name_premium` test

Reply:
You're correct. I forgot to save the Utils file even though I had deprecated the function there :sweat_smile: 

I've added it in another commit along with a fix for the test file.